ui/packages/statement/src/typings.d.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Default CSS definition for typescript,
+ * will be overridden with file-specific definitions by rollup
+ */
+declare module '*.css' {
+  const content: { [className: string]: string };
+  export default content;
+}
```

Comment:
![image](https://user-images.githubusercontent.com/1284531/76914219-8de31580-68f4-11ea-9a12-fd547c94f979.png)

If we remove this, we can't import the CSS file, and will build failed.

```
(plugin rpt2) Error: ...statement/src/components/StatementDetail.tsx(9,20): semantic error TS2307 Cannot find module './styles.module.css'.
```

Reply:
Your comment inspired me why I import yaml failed in .ts file before (I work around it by importing in js file as I told you before), because we don't declare the `*.yaml` module, it works fine after I adding this:

```ts
declare module '*.yaml' {
  const content: any;
  export default content;
}
```